kotlin-checks-test-sources/src/main/kotlin/checks/PropertyGetterAndSetterUsageCheckSample.kt

```diff
@@ -19,14 +21,172 @@ open class PropertyGetterAndSetterUsageCheckSample {
+        private var f5: Int = 0
+        internal fun getF5(): Int = f5 // Noncompliant
+        internal fun setF5(value: Int) { f5 = value }
+
+        // internal getter / private setter
+        private var f6: Int = 0
+        internal fun getF6(): Int = f6 // Noncompliant
+        private fun setF6(value: Int) { f6 = value }
+
+        // protected getter / protected setter
+        private var f7: Int = 0
+        protected fun getF7(): Int = f7 // Noncompliant
+        protected fun setF7(value: Int) { f7 = value }
+
+        // protected getter / private setter
+        private var f8: Int = 0
+        protected fun getF8(): Int = f8 // Noncompliant
+        private fun setF8(value: Int) { f8 = value }
+
+        // private getter / private setter
+        private var f9: Int = 0
+        private fun getF9(): Int = f9 // Noncompliant
+        private fun setF9(value: Int) { f9 = value }
+
+        // ______________________________________________
+        // 2) Setter with visibility > getter visibility
+
+        // internal getter / public setter
+        private var f10: Int = 0
+        internal fun getF10(): Int = f10 // Compliant
+        public fun setF10(value: Int) { f10 = value }
+
+        // internal getter / protected setter
+        private var f11: Int = 0
+        internal fun getF11(): Int = f11 // Compliant
+        protected fun setF11(value: Int) { f11 = value }
+
+        // protected getter / public setter
+        private var f12: Int = 0
+        protected fun getF12(): Int = f12 // Compliant
+        public fun setF12(value: Int) { f12 = value }
+
+        // protected getter / internal setter
+        private var f13: Int = 0
+        protected fun getF13(): Int = f13 // Compliant
+        internal fun setF13(value: Int) { f13 = value }
+
+        // private getter / public setter
+        private var f14: Int = 0
+        private fun getF14(): Int = f14 // Compliant
+        public fun setF14(value: Int) { f14 = value }
+
+        // private getter / internal setter
+        private var f15: Int = 0
+        private fun getF15(): Int = f15 // Compliant
+        internal fun setF15(value: Int) { f15 = value }
+
+        // private getter / protected setter
+        private var f16: Int = 0
+        private fun getF16(): Int = f16 // Compliant
+        protected fun setF16(value: Int) { f16 = value }
+
+        // missing getter / public setter
+        private var f20: Int = 0
+        fun setF20(value: Int) { f20 = value } // Compliant
+
+        // missing getter / internal setter
+        private var f21: Int = 0
+        internal fun setF21(value: Int) { f21 = value } // Compliant
+
+        // missing getter / protected setter
+        private var f22: Int = 0
+        protected fun setF22(value: Int) { f22 = value } // Compliant
+
+        // missing getter / private setter
+        private var f23: Int = 0
+        private fun setF23(value: Int) { f23 = value } // Compliant
```

Comment:
The field here has the same visibility as the setter - that means we don't need the setter.